libvast/include/vast/detail/legacy_deserialize.hpp

```diff
@@ -31,13 +31,21 @@
 #include <type_traits>
 #include <vector>
 
+#define CAF_VARIANT_ASSIGN_CASE_IMPL(n)                                        \
```

Comment:
This is a copy paste code from CAF. I believe it requires some annotation

Reply:
I think this is fine as-is if you move it to right above where you use it and then `#undef` it again immediately afterwards.